app/src/main/kotlin/com/softteco/template/ui/feature/home/HomeScreen.kt

```diff
@@ -1,59 +1,162 @@
+    val cards = listOf(
+        HomeCards.LoginCard(onLoginClicked),
+        HomeCards.SignatureCard(onSignatureClicked),
+        HomeCards.BleCard(onBleClicked),
     )
+    ScreenContent(cards)
 }
 
 @Composable
 private fun ScreenContent(
-    state: HomeViewModel.State,
+    cards: List<HomeCards>,
     modifier: Modifier = Modifier,
-    onApiSampleClicked: () -> Unit = {},
-    onGoLoggingClicked: () -> Unit = {}
 ) {
-    Box(modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
-        Column(verticalArrangement = Arrangement.spacedBy(Dimens.PaddingSmall)) {
-            Text(state.data)
-            Button(onClick = onApiSampleClicked) {
-                Text("To Api Sample")
-            }
-            Button(onClick = onGoLoggingClicked) {
-                Text(stringResource(id = R.string.go_to_login))
+    LazyVerticalGrid(
+        modifier = modifier,
+        columns = GridCells.Fixed(2),
+        contentPadding = PaddingValues(Dimens.PaddingSmall)
+    ) {
+        items(cards.size) { index ->
+            val item = cards[index]
+            HomeCard(
+                title = stringResource(id = item.titleId),
+                description = stringResource(id = item.descriptionId),
+                drawableRes = item.drawableRes,
+                onClick = item.onClick,
+                isEnabled = item.enabled,
+            )
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun HomeCard(
+    title: String,
+    description: String,
+    @DrawableRes drawableRes: Int,
+    modifier: Modifier = Modifier,
+    onClick: () -> Unit = {},
+    isEnabled: Boolean = true,
+) {
+    Card(
+        enabled = isEnabled,
+        modifier = modifier
+            .padding(Dimens.PaddingSmall),
+        onClick = { if (isEnabled) onClick() },
+        shape = RoundedCornerShape(8.dp),
+        colors = CardDefaults.cardColors(
+            containerColor = MaterialTheme.colorScheme.surfaceVariant,
+        )
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(Dimens.PaddingDefault),
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Column(modifier = Modifier.weight(1f)) {
+                Text(
+                    text = title,
+                    maxLines = 1,
+                    overflow = TextOverflow.Ellipsis,
+                    style = MaterialTheme.typography.titleMedium
+                )
+                Text(
+                    text = description,
+                    minLines = 2,
+                    maxLines = 2,
+                    overflow = TextOverflow.Ellipsis,
+                    style = MaterialTheme.typography.bodySmall,
+                )
             }
+            Icon(
+                modifier = Modifier.size(32.dp),
+                painter = painterResource(drawableRes),
+                contentDescription = null,
```

Comment:
please add a description, because you will receive a warning in the play console about this